### ISSUE TITLE
Changes to AJAX related aria-busy have caused keyboard nav failure

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#362]
A recent change to unsetting busy areas means that the busy phase is being removed after the post-insertion ajax subscribers have run. This causes keyboard walking of ajax menus to fail (and possibly others). 

We therefore need to be able to identify when the busy state has ended. @ricksbrown opinion?

**Comment by marksreeves** [comment]
Looks very much like we can work around the issue by wrapping the call to `setFocusRequest` in a zero timeout _and_ it only affected sub-menus which I have already fixed so I am closing this.
